### ISSUE TITLE
request function 

### ISSUE THREAD

**Issue by leogbar** [X-Ryl669/BwareArea#5]
This is a good app, the concept that use in the background with another GPS app is  excellent, but the mayor collections of pois is in gpx Garmin format, and import to the app is fundamentally, please can add this functionality

**Comment by X-Ryl669** [comment]
Yes, it's in my plans. Currently, I'm not happy with all the POI exchange format, they are all lacking some features in some way or another.

Since I've already added GPX export, adding GPX import should be possible without too much effort, I hope.

**Comment by X-Ryl669** [comment]
Here it is. Except a new release in F-Droid soon.

**Comment by X-Ryl669** [comment]
Since it's not appearing in F-Droid, you can find it in the release page. Cheers!

**Comment by leogbar** [comment]
Excellent! Soon I'll try it and I'll leave my comments !, I congratulate you for the application!

**Comment by leogbar** [comment]
The app is in fdroid, excellent for new functions, I probe with database gpx of 10000 POI's and tell you what happens!

**Comment by X-Ryl669** [comment]
Ok, good.

**Comment by leggewie** [comment]
what is the status here? @leogbar
